crispy_forms/bootstrap.py

```diff
@@ -386,3 +386,9 @@ def __init__(self, field, *args, **kwargs):
 
 class InlineField(Field):
     template = "%s/layout/inline_field.html"
+
+
+class ReadOnlyField(Field):
+    def __init__(self, *args, **kwargs):
+        self.attrs = {'readonly': 'readonly'}
```

Comment:
It sounds good to me if you are agree.

I will modify and send again.

Thanks.